USAGE_GUIDE.md

```
@@ -0,0 +1,118 @@
+
+- `dotbins.yaml` – Declarative list of managed tools and platform-specific options.
+- `linux/` & `macos/` – Platform folders that hold downloaded binaries organized by architecture.
+- `shell/` – Shell integration scripts. Source these from your shell configuration to expose dotbins-managed binaries and helper functions.
+- `configure-lfs-skip-smudge.py` – Helper script for configuring Git LFS in environments without automatic filtering.
+- `USAGE_GUIDE.md` – This document, intended to be updated as the repository evolves.
+
+## 2. Preparing Your Environment
+
+1. **Clone or update this repository** in your home directory so `$HOME/.dotbins` matches the expected location.
+2. **Install dotbins** if it is not already available:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)